overview.md

```diff
@@ -43,130 +43,173 @@ steps:
+- To download the PDF test report, follow the **1 published** link. 
+- To view the test results in JUnit-style XML format, open the **Tests** tab.
+- To view the code coverage results in Cobertura XML format, open the **Code Coverage** tab.
 
 
 ### Run MATLAB Script
-Use the [Run MATLAB Command](#run-matlab-command) task to run MATLAB scripts, functions, and statements. You can use this task to flexibly customize your test run or add a step in MATLAB to your pipeline. 
-
-For example, author a pipeline to run the commands in a file named `myscript.m`.
+Use MATLAB R2023b on a Microsoft-hosted agent to run the commands in a file named `myscript.m` in the root of your repository. To install the specified release of MATLAB on the agent, specify the **Install MATLAB** task in your pipeline. To run the script, specify the **Run MATLAB Command** task.
 
 ```YAML
-pool: myPool
+pool:
+  vmImage: ubuntu-latest
 steps:
+  - task: InstallMATLAB@1
+    inputs:
+      release: R2023b
   - task: RunMATLABCommand@1
     inputs:
       command: myscript
-``` 
+```
 
-### Specify MATLAB in Pipeline
-When you use the **Run MATLAB Build**, **Run MATLAB Tests**, or **Run MATLAB Command** task in your pipeline, the self-hosted agent uses the topmost MATLAB version on the system path. The pipeline fails if the agent cannot find any version of MATLAB on the path.
+### Specify MATLAB Release on Self-Hosted Agent
+When you use the **Run MATLAB Build**, **Run MATLAB Tests**, or **Run MATLAB Command** task in your pipeline, the agent uses the topmost MATLAB release on the system path. The task fails if the agent cannot find any release of MATLAB on the path.
 
-You can prepend your preferred version of MATLAB to the `PATH` environment variable of the agent. For example, prepend MATLAB R2023b to the path and use it to run your script.
+You can prepend your preferred release of MATLAB to the `PATH` system environment variable of the self-hosted agent. For example, prepend MATLAB R2020b to the path and use it to run your script. The step depends on your operating system and MATLAB root folder.
 
 ```YAML
 pool: myPool
 steps:
-  - powershell: Write-Host '##vso[task.prependpath]C:\Program Files\MATLAB\R2023b\bin'  # Windows agent
-# - bash: echo '##vso[task.prependpath]/usr/local/MATLAB/R2023b/bin'  # Linux agent
+  - powershell: Write-Host '##vso[task.prependpath]C:\Program Files\MATLAB\R2020b\bin'  # Windows agent
+# - bash: echo '##vso[task.prependpath]/usr/local/MATLAB/R2020b/bin'  # Linux agent
   - task: RunMATLABCommand@1
     inputs:
       command: myscript
 ```
 
-### Use MATLAB on Microsoft-Hosted Agent
-Before you run MATLAB code or Simulink models on a Microsoft-hosted agent, first use the [Install MATLAB](#install-matlab) task. The task installs your specified MATLAB release (R2021a or later) on a Linux&reg; virtual machine. If you do not specify a release, the task installs the latest release of MATLAB.
+### Use MATLAB Batch Licensing Token
```

Comment:
Should we add anything about the fact that batch licensing tokens require a certain version of the extension and above? So if they are using v0 currently they need to use v1?

Reply:
I don't think it will be necessary because we only maintain documentation for the most recent version. Also, we issue warnings for those who use v0 of the extension and encourage them to upgrade :) 

Example:
"The specified version of the MATLAB extension is no longer supported. Update your pipeline to use the most recent version of the extension."